src/test/java/edu/umich/lib/dor/replicaexperiment/DepositTest.java

```diff
@@ -121,7 +121,7 @@ void depositExecutes() {
         when(repositoryServiceMock.getRepository("some_repo")).thenReturn(repositoryMock);
         when(registryMock.getClient("some_repo")).thenReturn(clientMock);
 
-        var deposit = depositFactory.create(
+        final var deposit = depositFactory.create(
```

Comment:
I mentioned I like to declare things final. I should clarify slightly:

- I like to declare *collaborators* and effective constants as final for regular classes (and usually set collaborators in the constructor)
- I like to define *constants* as final in test classes (and usually set them statically at the declaration)

The stuff that should be built/rebuilt for the suite or individual tests, I leave as locals or regular, reassignable instance variables.

Reply:
Good to know. I added this here because the linter was complaining that it was too far away from later references (because of the all the stubbing), and this solved it. I can just ignore the warning.

Reply:
Oh, interesting! That's kind of cool that the linter has a threshold for proximity... So, it serves the same kind of purpose I mentioned. It's easy to see an intentional or incidental mutation a few lines away. It's trickier to see the violation of intention a screen or two away.

No problem with the final on the local!